examples/website/i3s-building-scene-layer/src/helpers/sublayers.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * - deep copy the tree for React
+ * - filter "preview" layer
+ * - calculate the number of child nodes for each node (just in case)
+ * @param {*} sublayers
+ * @returns
+ */
+export function buildSublayersTree(sublayers) {
```

Comment:
Possible to convert to .ts and add types?

Reply:
Sure, done